react/week1/to_do_list/src/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+const tasks = [
+  { title: "Do HYF homework", deadline: "2020-03-25", color: "purple" },
+  { title: "Wash the dishes", deadline: "2020-03-25", color: "orange" },
+  { title: "run the washing machine", deadline: "2020-03-25", color: "blue" },
+  { title: "go for shopping", deadline: "2020-03-25", color: "green" }
+];
+
+const headerText = "My ToDo List";
+
+ReactDOM.render(
+  <div>
+    <Header text={headerText} />
+    <ListMaker mytasks={tasks} />
+  </div>,
+  document.getElementById("root")
+);
+
+function Header(props) {
+  return <h1>{props.text}</h1>;
+}
+
+function ListMaker(props) {
```

Comment:
You should name your components based what it renders instead of how it renders elements. I would consider renaming this to `TodoList` instead. One of the nice parts about writing code in react as components, is it makes it much easier to describe the exact view the users sees in code e.g.

```jsx
<body>
    <Header />
    <TodoList />
    <Footer />
</body>
```

One can easily see that the user sees a list of todos with along with a header and a footer. Contrast this against: 

```jsx
<body>
    <Header />
    <ListMaker />
    <Footer />
</body>
```

Someone reading the code can figure out that a list is being rendered, however it isn't entirely clear what type of list is being shown. It's difficult to see the value in this now with such concise examples but I promise you it will become more clear as we continue on!